### ISSUE TITLE
LayoutGrids.html example's documentation for "aria-label" incorrect

### ISSUE THREAD

**Issue by spectranaut** [w3c/aria-practices#751]
This is small bug in the documentation for the [grid/LayoutGrids.hml example page](https://www.w3.org/TR/wai-aria-practices-1.1/examples/grid/LayoutGrids.html). In the "Role, Property, State, and Tabindex Attributes" table, the row for `aria-label="Search Results"` contains a usage description: `In example 3, the grid does not have a visible label so the label is specified with aria-label.`

However, example 3 actually does has a visible label, and, appropriately, uses `aria-labelledBy` to indicate which label to use.

I see two possible fixes:
(1) Removes the `aria-label` row in this table (as there is no example that uses it)
(2) Updates example three to not have a visible label and use `aria-label` on the grid element.

**Comment by mcking65** [comment]
Fixed with commit 6c93a35.
